### PR TITLE
DAMIEN-320: Fixed announcement banner font, links

### PR DESCRIPTION
**Pull request by cesarvh**
https://jira-secure.berkeley.edu/browse/DAMIEN-320

### CODE REVIEW

File: src/views/BaseView.vue [hunk 1]
```diff
@@ -208,5 +209,6 @@
 }
 pre {
   white-space: pre-line;
+  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
```
Comment (lyttam) [inline src/views/BaseView.vue]: @cesarvh If you add `class="body-2"` to the `<pre>` element, you won't need to hard-code the font-family here.

File: src/assets/styles/damien-global.css [hunk 2]
```diff
@@ -184,4 +184,7 @@ svg {
 }
 ul {
   list-style: none;
-}
\ No newline at end of file
+}
+#content a {
+  color: #337ab7;
+}
```
Comment (lyttam) [inline src/assets/styles/damien-global.css]: @cesarvh The color applied to all `<a>` elements is configured in vuetify.ts - see `themes.light.anchor` and `themes.dark.anchor`. Currently it's set to a nice shade of orange - which probably won't look good on the yellow background of the service alert. 

Maybe make your CSS selector here more specific, so it only overrides the configured anchor color inside the service alert element.

File: src/views/BaseView.vue [hunk 3]
```diff
@@ -186,7 +188,7 @@
   }
 </script>
 
-<style scoped>
+<style scoped override>
```
Comment (pauline2k) [inline src/views/BaseView.vue]: @cesarvh Interesting, I haven't seen this `override` tag before. Where did you find it documented?

Comment (cesarvh) [reply src/views/BaseView.vue]: I forgot where I found it, but I can't seem to find it again. This was back when I was working on the printable view for the degree progress. Removed!

**Comment by pauline2k** [comment]
Thanks @cesarvh!

**Pull request merged**